src/search/sd-record-list.js

```diff
@@ -90,7 +90,7 @@ class SdRecordList extends GestureEventListeners(PolymerElement) {
     goToDeeplink(e) {
         console.log(e);
         if (e.model.__data.item.record.deeplink !== "") {
-            window.open(e.model.__data.item.record.deeplink, "_blank")
+            window.open(window.location.domain + e.model.__data.item.record.deeplink, "_blank")
```

Comment:
Can't reproduce the fix on local machine.
Opens a URL like this: `undefinedhttps://web-components.fact-finder.de/api/ff-template#tab=docs`.
`e.model.__data.item.record.deeplink` returns an absolute URL.
What is `domain` supposed to be? `origin` perhaps?
```
window.location = {
  "href": "http://localhost:8081/search?query=asn&channel=webc-doku-text&sid=...",
  "ancestorOrigins":{},
  "origin": "http://localhost:8081",
  "protocol": "http:",
  "host": "localhost:8081",
  "hostname": "localhost",
  "port": "8081",
  "pathname": "/search",
  "search": "?query=asn&channel=webc-doku-text&sid=...",
  "hash": ""
}
```

Reply:
You're right, domain isn't always defined. `host` would be more suitable